### ISSUE TITLE
Restart a turbinesFoam simulation

### ISSUE THREAD

**Issue by kkpal3** [turbinesFoam/turbinesFoam#320]
Hello @petebachant,
When I restart a turbinesFoam simulation from a stopped one, there will be large oscillations in the beginning of the restart instant, even with first-order Euler scheme. I guess there might be some forcing information that are not written to the file?
Could you suggest which part of the code to modify to solve this restart problem?
Many thanks!

**Comment by petebachant** [comment]
That's correct that the code is not setup for restarts, but I believe the information should be written to achieve this. The turbine state (azimuth, etc) is written to a `postProcessing` directory, so that would need to be read to set the azimuth and angular velocity. That would happen in the turbine classes. Then the force field would need to be loaded for the start time in the actuator line class.
